packages/mui-material/package.json

```diff
@@ -37,7 +37,8 @@
     "release": "pnpm build && pnpm publish",
     "test": "cd ../../ && cross-env NODE_ENV=test mocha 'packages/mui-material/**/*.test.?(c|m)[jt]s?(x)'",
     "typescript": "tsc -p tsconfig.json",
-    "typescript:module-augmentation": "node scripts/testModuleAugmentation.js"
+    "typescript:module-augmentation": "node scripts/testModuleAugmentation.js",
+    "attw": "attw --pack ./build --exclude-entrypoints esm modern --include-entrypoints Button styles"
```

Comment:
Should we add the `attw` check for all packages?

Why exclude esm and modern?

Reply:
I added a few key packages that most people would use, and the icons that have a more exotic setup. didn't want to add too much to CI for packages that most people won't import and that won't regress on their own anyway as it all uses the same automation. We can remove the exclusions after https://github.com/arethetypeswrong/arethetypeswrong.github.io/issues/242 is fixed.